rules/sinks/third_parties/sdk/starkbank/php.yaml

```diff
@@ -9,5 +9,5 @@ sinks:
     domains:
       - "starkbank.com"
     patterns:
-      - "(?i)(starkbank).*"
+      - "(?i)(starkbank|EllipticCurve).*"
```

Comment:
??

Reply:
removed it, as its a signing algo library.